### ISSUE TITLE
Assertion failed: (fd > STDERR_FILENO)

### ISSUE THREAD

**Issue by derhuerst** [0x00A/prompt-sync#14]
When doing `if (null === prompt.prompt({ask: 'foo?'})) process.exit(0)`, I get:

```
Assertion failed: (fd > STDERR_FILENO), function uv__close, file ../deps/uv/src/unix/core.c, line 487.
```

**Comment by davidmarkclements** [comment]
This may be fixed in master but not released yet - @derhuerst would you mind cloning and checking abainst master?

**Comment by 0x00A** [comment]
@davidmarkclements added you as a owner on npm (`+ davidmarkclements (prompt-sync)`)

**Comment by davidmarkclements** [comment]
cool thanks :bowtie: - @derhuerst once you confirm I'll publish (or co-debug, fix then publish if you find it's still not working your end)

**Comment by davidmarkclements** [comment]
we're about to go to v4.0.0 - we should still publish a fix for v3 but want also want to make sure it works on @derhuerst system

**Comment by davidmarkclements** [comment]
v4 published and 3.0.3 which should hopefully contain fix - closing
